setup.py

```diff
@@ -32,7 +32,7 @@
 
 setup(
     name='google-auth',
-    version='1.6.3',
+    version='1.6.4',
```

Comment:
We cut separate release PRs to just bump the version and add changelog notes. See #325 for an example.

Reply:
OK, sorry, did not know that. I will revert the version to the one present before.

Reply:
I squashed the commits in order to have a single commit with the fix (thus avoiding a commit just for reverting the version back, I do not know if this is compliant with your branch strategy). Let me know if there is other I can do. Thanks!